.github/workflows/ci.yml

```
@@ -1265,33 +1262,40 @@ jobs:
   # Not strictly required for omnibase_core (no path filtering on Tier A gates),
   # but added for uniformity across all OmniNode repos.
 
-  contract-compliance:
+  compliance:
```

actionlint (1.7.12)

[error] key "compliance" is duplicated in "jobs" section. previously defined at line:735,col:3. note that this key is case insensitive

(syntax-check)